### ISSUE TITLE
Theme Demo: Improve purchase bar

### ISSUE THREAD

**Issue by dsas** [Automattic/wp-calypso#85539]
Theme demo sites e.g. https://peiraodemo.wordpress.com/ display a pricing banner across the top of the site. The aim is to help people use this theme on their site, or create a new site with this theme.

![Image](https://github.com/Automattic/wp-calypso/assets/93301/72d2c00d-31eb-4894-bb91-46491f626cef)

These banners used to more prominent, but now they're only usually seen when we link directly to them, like in our regular [new themes blog posts](https://wordpress.com/blog/tag/themes/).

We've recently made some quick changes to simplify the old, complex code behind the banners (D131004-code) and now we need to refine it to make a clearer CTA. See p1702984559032929-slack-C048GFLMBGD

Changes also need to be duplicated 😒 for demo sites that are hosted on our atomic infrastructure see https://github.com/Automattic/dotcom-forge/issues/4856#issue-2038090926 for some info. It's fine to do that as a follow-up issue.

If re-using the current design/assets then it's important that we test with Safari because there's a bug reported  with the logo appearing to be vertically stretched in comparison to the assets at https://dotcombrand.wordpress.com/logotype/ (tho I can't see it).

**Comment by itssachinkr** [comment]
I would like to work on this issue, can you please assign it to me?

**Comment by itssachinkr** [comment]
so what i understood is, we just have to remove that wordpress theme price from the bar?

**Comment by dsas** [comment]
@pseudofool, thanks for offering. The theme price has already been removed from the bars, we need to figure out what the bar should look like to make a really obvious and compelling call-to-action to users.

**Comment by yashghogre** [comment]
Is this issue still open?

**Comment by dsas** [comment]
@yashghogre yes it's open, as in it is not complete. It's assigned to Matt who is going to figure out a design.

**Comment by leynarhuynh** [comment]
Hi! I'm new to open source contribution but would love to give this issue a shot.
